backend/src/main/java/com/staccato/category/service/CategoryService.java

```diff
@@ -0,0 +1,133 @@
+package com.staccato.category.service;
+
+import com.staccato.category.domain.Category;
+import com.staccato.category.repository.CategoryMemberRepository;
+import com.staccato.category.service.dto.request.CategoryReadRequest;
+import com.staccato.category.service.dto.request.CategoryRequest;
+import com.staccato.category.service.dto.response.CategoryDetailResponse;
+import com.staccato.category.service.dto.response.CategoryIdResponse;
+import com.staccato.category.service.dto.response.CategoryNameResponses;
+import com.staccato.category.service.dto.response.CategoryResponses;
+import com.staccato.staccato.domain.Staccato;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import com.staccato.comment.repository.CommentRepository;
+import com.staccato.config.log.annotation.Trace;
+import com.staccato.exception.ForbiddenException;
+import com.staccato.exception.StaccatoException;
+import com.staccato.member.domain.Member;
+import com.staccato.category.domain.CategoryMember;
+import com.staccato.category.repository.CategoryRepository;
+import com.staccato.staccato.repository.StaccatoImageRepository;
+import com.staccato.staccato.repository.StaccatoRepository;
+import lombok.RequiredArgsConstructor;
+
+@Trace
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class CategoryService {
+    private static final List<CategoryFilter> DEFAULT_CATEGORY_FILTER = List.of();
+    private static final CategorySort DEFAULT_CATEGORY_SORT = CategorySort.UPDATED;
+
+    private final CategoryRepository categoryRepository;
+    private final CategoryMemberRepository categoryMemberRepository;
+    private final StaccatoRepository staccatoRepository;
+    private final StaccatoImageRepository staccatoImageRepository;
+    private final CommentRepository commentRepository;
+
+    @Transactional
+    public CategoryIdResponse createCategory(CategoryRequest categoryRequest, Member member) {
+        Category category = categoryRequest.toCategory();
+        validateCategoryTitle(category, member);
+        category.addCategoryMember(member);
+        categoryRepository.save(category);
+        return new CategoryIdResponse(category.getId());
+    }
+
+    public CategoryResponses readAllCategories(Member member, CategoryReadRequest categoryReadRequest) {
+        List<Category> rawMemories = getCategories(categoryMemberRepository.findAllByMemberId(member.getId()));
```

Comment:
```suggestion
        List<Category> rawCategories = getCategories(categoryMemberRepository.findAllByMemberId(member.getId()));
```

아직 대문자로 시작하는 `Memories` 들은 몇몇개 반영 안 되어 있습니다. 요것들만 추가로 반영해주세요! 😄